### PR TITLE
Epo 515

### PR DESCRIPTION
**Pull request by dbd311**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -310,3 +310,6 @@ TSWLatexianTemp*
 *.synctex.gz
 *.xpr
 /data/
+
+xpec/.git
+xpec/.github/*
```
Comment (costezki) [inline .gitignore:315]: ant-contrib.jar and saxon...jar shall not be comited to GitHub

File: test/README.txt.txt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+To run the unit tests:
+
+$ cd ${project.dir}
+$ ant -lib lib/saxon-he-10.6.jar   unit_tests
```
Comment (costezki) [inline test/README.txt.txt:4]: 1. the `readme file` name is Readme.md, please rename. 
2. the readme file shall be ONLY ONE and in the project root.

Comment (costezki) [reply test/README.txt.txt:4]: We need a Makefile that 
1. install the necessary libraries
2. runs tests

File: xspec/.codespellrc [hunk 3]
```diff
@@ -0,0 +1,8 @@
+[codespell]
```
Comment (costezki) [inline xspec/.codespellrc:1]: why do we need these files? 
Please delete the 4 `.xxx` files. In the future these shall be (Git) ignored if necessary only in the dev environment.

File: xspec/assembly/enduser-files.xml [hunk 4]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (costezki) [inline xspec/assembly/enduser-files.xml:1]: what is the role of this folder? 
I see many technical folders and files at the top level of the repo. Can we hide them somehow?

**Review by costezki — CHANGES_REQUESTED** [review]

**Comment by dbd311** [comment]
It was discussed with Dragos as well. I will close the PR and write a make
file which define different tasks running automatic tests.

File: xspec/.codespellrc (hunk 3 above)
Comment (costezki) [reply xspec/.codespellrc:1]: do we need XSPEC folder? 
I start doubting this. What is its purpose?

**Comment by dbd311** [comment]
Xspec is required but I will put it in the make file

**Pull request closed without merging**